Euroleague-Graduate/Euroleague-Graduate/View/TurkishLeagueFFTableViewController.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  TurkishLeagueF4TableViewController.swift
+//  Euroleague-Graduate
+//
+//  Created by Modeso on 7/24/17.
+//  Copyright © 2017 Modeso. All rights reserved.
```

Comment:
TurkishLeaguePOTableViewController and TurkishLeagueFFTableViewController, TurkishLeagueRegularSeosonTableViewController 

they have the same code... 
can't u Just use one view controller
let's say 
MatchListViewController... and change round only in it?